test/e2e/app-dir/app-prefetch/prefetching.test.ts

```diff
@@ -331,6 +331,16 @@ describe('app dir - prefetching', () => {
           0
         )
       })
+
+      // navigate to the static page
+      await browser.elementByCss('[href="/static-page"]').click()
+
+      // We still shouldn't see any requests
+      await retry(async () => {
+        expect(rscRequests.filter((req) => req === '/static-page').length).toBe(
+          0
+        )
+      })
```

Comment:
Should we add an expectation on the static page content between click and checking the rsc requests, to ensure that the page has finished rendering, and thus avoiding a false positive rsc requests check, in case this ever regresses?